Dockerfile

```diff
@@ -36,7 +36,9 @@ RUN chown hadoop /opt
 ADD scripts /opt/
 ADD scripts/krb5.conf /etc/
 RUN yum install -y krb5-workstation
-
+RUN mkdir -p /etc/hadoop && mkdir -p /var/log/hadoop && chmod 777 /etc/hadoop && chmod 777 /var/log/hadoop
+ENV HADOOP_LOG_DIR=/var/log/hadoop
+ENV HADOOP_CONF_DIR=/etc/hadoop
```

Comment:
Did you want to write /etc/ozone here ?

Reply:
This is a hadoop-runner image I can't write ozone. But we can introduce a separated apache/ozone-runner where we can use ozone names inside. (And we need to fix hadoop-runner anyway)

Reply:
We must stop the apache/*-runner idea.  It is a huge time sink for development.  It has also been noted *-runner design broke/by pass Apache voting and release process for Ozone 0.4.0 release.  Let's not go there again.

Reply:
Thanks @wyanghwx the comment. I  think your opinion is under discussion at https://issues.apache.org/jira/browse/HDDS-1495 and personally I am convinced yet.

But this patch doesn't introduce any new method just fixes a small bug in the existing practice (which is not preferred by you)

Do you prefer to stop all the bugfixing until we got an agreement in HDDS-1495? Do you agree to commit this patch because it's required in the current situation? Do you see any technical problems with applying this patch?


ps: here is a comment: 
https://lists.apache.org/thread.html/e5c17f08f30aa924bdfd50d215b3f468269dc9198411b069d035a891@%3Cgeneral.incubator.apache.org%3E

> The main Docker Hub at hub.docker.com is a public-facing downstream
distribution channel -- similar to Maven Central, PyPI, Debian package
management, etc.

> It is appropriate to distribute official releases through downstream channels,
but inappropriate to distribute unreleased materials through them.  (That's
why having `latest` on hub.docker.com point to git `master` is problematic.)

This is what we are doing right now. We are distributing existing releases via downstream channels.

And finally, I kindly ask to continue the discussion about the current docker practices in HDDS-1495 and don't start the same conversation everywhere. I propose it just because it can be easier to follow and manage. Does it work for you?

Reply:
+1, Sorry for the confusion. I misunderstood the patch. Let us commit this.

Reply:
chmod 777 is a security hole.  It should have sticky bit set (1777) to ensure that normal users can not delete file written by root or other users.

Reply:
Thanks the comment @eyanghwx. I added the sticky bit in the latest commit.

Reply:
+1, I will commit this now.